DESCRIPTION

```diff
@@ -1,6 +1,5 @@
 Package: EpiEstim
-Version: 2.1-0
-Date: 2012/01/17
+Version: 2.0-0
```

Comment:
Version is going backwards..?

Reply:
sorry, that was a mistake of merging the files from different branches.